build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
 }
 
 group 'io.cify'
-version '1.3.26'
+version '1.4.1'
```

Comment:
Current version we have in HERE Gerrit is 1.4.1 and now we are merging both repos (HERE and github). I  guess . it makes sense to increase a version, maybe at least to 1.4.2

Reply:
Increased version to 1.4.2